addons/crm/models/res_partner.py

```diff
@@ -35,14 +35,18 @@ def default_get(self, fields):
                 )
         return rec
 
-    def _compute_opportunity_count(self):
+    def _get_all_partners(self):
```

Comment:
Naming could probably be improved, like
```suggestion
    def _fetch_children_partners_for_hierarchy(self):
```